### ISSUE TITLE
Few UI bugs / suggestions

### ISSUE THREAD

**Issue by louneskmt** [L2-Technology/sensei#60]
Here are a few UI bugs to fix, or suggestions. This is a good first issue if you wanna start contributing to Sensei!

1) Some table views have an horizontal scroll while they shouldn't have one
<img width="600" alt="image" src="https://user-images.githubusercontent.com/38882571/164792907-d68e58a7-ad1f-4f62-b049-7fe6ceeac4f6.png">
<img width="600" alt="image" src="https://user-images.githubusercontent.com/38882571/164792957-4c74ba01-855f-4bc8-b0e0-0a6c7657cb83.png">

Expected behaviour:

<img width="600" alt="image" src="https://user-images.githubusercontent.com/38882571/164793201-bbb091ba-248f-47c9-96c4-7a9a732423a6.png">

2) Three dots menu positions are not relative to the three dot button, but to the border of the page

<img width="300" alt="image" src="https://user-images.githubusercontent.com/38882571/164793376-73b346bf-9a6a-4301-9ee7-f2d7393ce3c9.png"> <img width="300" alt="image" src="https://user-images.githubusercontent.com/38882571/164793410-050f99d3-0d45-4b51-90c6-307947dd2a36.png">

3) Invoice amount input shouldn't accept negative values
4) Invoice amount should be in sats

<img width="486" alt="image" src="https://user-images.githubusercontent.com/38882571/164793618-2b9923cb-8529-40d5-b7ce-d2074dcbc6a6.png">

5) Loading bar is broken

<img width="1004" alt="image" src="https://user-images.githubusercontent.com/38882571/164794067-4ee75430-7783-4ba7-a3f1-752c0b662ab2.png">

6) Add Sensei logo as favicon

<img width="201" alt="image" src="https://user-images.githubusercontent.com/38882571/164794115-3f954115-a76b-4351-b44e-29cc14dad4f5.png">

7) Move "Fund Node" content to "Chain" tab?

**Comment by nassersaazi** [comment]
I would like to help with this @louneskmt .however am having some trouble setting up the Sensei node in my local environment 

I am gettting the following error: 

"```
thread 'main' panicked at 'invalid bitcoind rpc config: Os { code: 111, kind: ConnectionRefused, message: "Connection refused" }', src/main.rs:182:10
note: run with RUST_BACKTRACE=1 environment variable to display a backtrace
```"


that is after running this command:
`cargo run --bin senseid -- --network=regtest --bitcoind-rpc-host=localhost --bitcoind-rpc-port=18443 --bitcoind-rpc-username=admin1 --bitcoind-rpc-password=123`

any help is highly appreciated 🙂

**Comment by johncantrell97** [comment]
fixed the rest of these in #91
